project-base/storefront/components/Blocks/Categories/PromotedCategories.tsx

```diff
@@ -1,17 +1,25 @@
-import { SimpleNavigation } from 'components/Blocks/SimpleNavigation/SimpleNavigation';
+import { PromotedCategoriesContent } from './PromotedCategoriesContent';
 import { SkeletonModulePromotedCategories } from 'components/Blocks/Skeleton/SkeletonModulePromotedCategories';
+import { Webline } from 'components/Layout/Webline/Webline';
 import { usePromotedCategoriesQuery } from 'graphql/requests/categories/queries/PromotedCategoriesQuery.generated';
+import useTranslation from 'next-translate/useTranslation';
 
 export const PromotedCategories: FC = () => {
+    const { t } = useTranslation();
     const [{ data: promotedCategoriesData, fetching: arePromotedCategoriesFetching }] = usePromotedCategoriesQuery();
 
-    if (arePromotedCategoriesFetching) {
-        return <SkeletonModulePromotedCategories />;
-    }
-
-    if (!promotedCategoriesData) {
+    if (!promotedCategoriesData?.promotedCategories.length) {
         return null;
     }
 
-    return <SimpleNavigation listedItems={promotedCategoriesData.promotedCategories} />;
+    return (
+        <Webline className="mb-10">
+            <h3 className="mb-4">{t('Shop by category')}</h3>
+            {arePromotedCategoriesFetching ? (
+                <SkeletonModulePromotedCategories />
+            ) : (
+                <PromotedCategoriesContent promotedCategoriesData={promotedCategoriesData} />
+            )}
+        </Webline>
+    );
```

Comment:
Are you sure this still shows the skeleton loader?

Reply:
fixed